src/scripts/tbtcv2-rewards/rewards-constants.ts

```diff
@@ -1,7 +1,7 @@
-// 3 weeks in sec (default value) + we add 6 days becuase the -m5 release was
-// announced 6 days after the tag was created. This applies only for the October
-// rewards interval.
-export const ALLOWED_UPGRADE_DELAY = 2332800
+// 3 weeks in sec (default value) + we add 20 days because the -m6 release was
+// announced 20 days after the tag was created. This applies only for the October
+// and November rewards interval (cutoff date for -m6 is Nov 15).
+export const ALLOWED_UPGRADE_DELAY = 3542400
```

Comment:
```suggestion
// 3 weeks in sec (default value) + we add 22,67 days because the -m6 release was
// announced 22,67 days after the tag was created. This applies only for the October
// and November rewards interval (cutoff date for -m6 is Nov 15).
export const ALLOWED_UPGRADE_DELAY = 3773887
```

Reply:
Your calculations are proper! I initially used the date from the release notes (https://github.com/keep-network/keep-core/releases/tag/v2.0.0-m6) assuming the tag was created the same day. However, the initial release notes were removed and then added again 2-3 days later (we had to delay publication for a while), hence the drift.